libs/damap/src/lib/components/dmp/dmp-actions/dmp-actions.component.ts

```diff
@@ -97,15 +97,16 @@ export class DmpActionsComponent implements OnInit, OnDestroy {
 
   exportDmpTemplate(): void {
     const dialogRef = this.dialog.open(ExportWarningDialogComponent, {});
-    dialogRef.componentInstance.funderSupported =
-      this.dmpForm.controls.project.getRawValue().funderSupported;
+    let funderSupported: boolean =
+      this.dmpForm.controls.project?.getRawValue()?.funderSupported ?? false;
```

Comment:
Project or rawValue may not be set, so we gate access.
Also refactor to only access once.